packages/mui-component-mapper/src/date-picker/date-picker.d.ts

```diff
@@ -6,6 +6,10 @@ type InternalDatePickerProps<TInputDate, TDate> = MuiDatePickerProps<TInputDate,
 }
 
 export type DatePickerProps<TInputDate, TDate> = InternalDatePickerProps<TInputDate, TDate> & UseFieldApiComponentConfig;
+<<<<<<< Updated upstream
```

Comment:
@Hyperkid123 Should these be removed? I think these are the git comments but might cause parsing errors.

Reply:
Oh yeah. Nice catch!